### ISSUE TITLE
3d object with rigged (Skeletal animation) doesn't work

### ISSUE THREAD

**Issue by fraviloria** [viromedia/viro#677]
**Environment**

Development OS: Mac
Version: ViroReact version: 2.15.0
React Native version: 0.60.1
Device(s): iPhone X, Xs Max, Xr, 6s plus, 7 iOS 12.3.1,  Samsung s8, s8+, Note 8

The 3d object (GLTF and GLB) with rigged (Skeletal animation) doesn't work on ViroReact.

This is an example of the 3d object on GLTF (https://drive.google.com/file/d/1IFeCPc01BAoK4hxFC2Ow-4ULpoi3sKmm/view?usp=sharing)

**Comment by Turtleted21** [comment]
yes animated models glTF are not supported yet, you can use fbx.

**Comment by fraviloria** [comment]
@Turtleted21 the gLTF animated is work but my issue is gLTF with rigged (Skeletal animation).

@dam00n @radvani Please help me to verify this issue.

**Comment by dthian** [comment]
Hey @fraviloria, thanks for reporting this, we are currently taking a look into this issue now.

**Comment by BMR11** [comment]
Hey @dthian , If you would like to get more detail about similar crash on android. 

I am also trying to do same thing like playing gltf animation 
I have posted crash log 
https://github.com/viromedia/viro/issues/676

**Comment by fraviloria** [comment]
@BMR11  My gLTF load on Viro Camera isn't crashing, but without movement I mean static, 
@dthian is helping me about it

**Comment by BMR11** [comment]
Thanks @fraviloria for details but I am having same issue like my gltf animation is a rigged skeleton animation and I can preview it using https://gltf-viewer.donmccurdy.com/ properly.

Though I will create another issue in case your case may be different from mine.

**Comment by fraviloria** [comment]
@BMR11 that's correct, the gLTF work properly using this page (https://gltf-viewer.donmccurdy.com/),
for that reason, I think that the problem is with ViroReact.

@dthian Do you have any update with it?

**Comment by Turtleted21** [comment]
ok I understand why animated model from Sketchfab don't work, because these are Skeletal animation.
Tested this Turtle model:
https://skfb.ly/6HGqp
Don't work ViroReact, work with viewer donmccurdy

**Comment by Turtleted21** [comment]
don't work for me, sometimes no animation sometimes only few bones are animated

**Comment by fraviloria** [comment]
Hi @dthian with have the same issue, after the upgrade (v2.16.0) the GLTF with Skeletal animation load deformed or only few part are animated.

**Comment by dthian** [comment]
Hey all, the v.2.16.0 upgrade fixed some skeletal animation issues but not all. This is a known bug. I'll update this bug soon as we've root caused and determined a fix.

**Comment by antoinerousseau** [comment]
Is RN 0.60 supported anyway?

**Comment by parul1986** [comment]
Hi @dthian , I also have the same issue, is the gltf loading with skeletal animation issue fixed, any updates on this.

**Comment by mstrokin** [comment]
@dthian Any update on this?

**Comment by dthian** [comment]
Hey @mstrokin have you verified if your model works with the latest build of viro? If it doesn't, unfortunately we are not providing dedicated support at this time, and have open-sourced the [renderer](https://github.com/viromedia/virocore), as [mentioned](https://blog.viromedia.com/viro-is-going-open-source-be9da0b43328). 

Please feel free to take a stab at the bug, or reach out to the community in our Slack channels.

**Comment by mstrokin** [comment]
@dthian I haven't, sorry, we're using FBX-VRX files but I was researching if it's possible to swith to glTF and stumbled upon this issue (because we're using skeletal animation as well)

Will try to find spare time to test this next week.

**Comment by tellypresence** [comment]
Very nice how simple it is to load gltf files (just need to make sure textures and .bin files co-located in same relative folder location with the gltf in assets)

Unfortunately gltf skeletal animation still unsupported, seems from the comments above that further development may have been suspended
```
Viro: Removing mis-matching animation - layered skeletal animations not yet supported for gLTF.
```

**Comment by dthian** [comment]
Actually, gltf skeletal animations are supported, just not layered ones unfortunately. 
It might be supported in FBX models though.

**Comment by byron-the-bulb** [comment]
We are currently blocked by this issue and wondering if there are any updates, or if others have been able to get around it while still using gLTF ( FBX is not an easy option for us) ?
What exactly is meant by "layered animations" in this context? Does it refer to the usage of gLTF "skins" as defined in the specs? https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#skins 

 Versions:
 "@viro-community/react-viro": "^2.23.0",
 "react": "17.0.2",
 "react-native": "0.68.2"
